libs/blog/articles/data-access/src/lib/guards/article-exists.guard.ts

```diff
@@ -45,7 +45,7 @@ export const articleExistsGuard: CanActivateFn = (route) => {
               {},
             );
           } else {
```

Comment:
maybe add a comment like: // otherwise redirect to the home page in the requested language

Reply:
Appended